src/styles/List.js

```diff
@@ -20,13 +20,28 @@ const Container = styled.ul`
     background-color: ${colors.lightestGrey};
   }
 
-  .participants {
-    height: 15px;
-    border-radius: 10px;
+  .avatar-stack {
     vertical-align: middle;
-    margin-right: 0;
-    margin-left: 5px;
-    margin-top: 5px;
+    display: inline;
+    margin-left: 3px;
+
+    .avatar {
+      height: ${size.tiny};
+      margin-right: -${size.micro};
+      margin-top: 10px;
+      background-color: white;
+      border: 1px solid white;
```

Comment:
If the background color is already set to white, do we need to put border as well? I think the border is not needed.

Reply:
@sujitagarwal The images seem merged without a border!

| Without Border | With Border |
|---|---|
| ![Screenshot](https://user-images.githubusercontent.com/61746440/85026646-8bfc4d00-b196-11ea-81a5-f2544ba0d414.png) | ![Screenshot](https://user-images.githubusercontent.com/61746440/85026695-9d455980-b196-11ea-8f32-6ab654025401.png) |

Reply:
with-border matches what GitHub does as well.

Reply:
I see... In this scenario it is fine then...